docs/TROUBLE.md

```diff
@@ -32,27 +32,27 @@ Carefully install your Tx module into your OpenTX or EdgeTX radio.  Some Tx modu
 
-When custom firmware with the same bind phrase or the pre-compiled firmware is installed on new or freshly erased Receiver and Tx modules, they should connect as they will already be bound with the same bind phrase.
+When custom firmware with the same bind phrase or the pre-compiled firmware is installed on new or freshly erased Tx modules and receivers, they should connect as they will already be bound with the same bind phrase.
 
-Power up a single Tx module and compatible Rx.
+Power up a single Tx module and compatible receiver.
 
 When the Tx module and receiver are bound and powered on, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
 
 You should also use the [mLRS Lua script](LUA.md) to confirm the radio link and firmware versions.  It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.
 
 - If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
 - If the binding procedure doesn't establish the connection, verify that both the receiver and Tx module are compatible, which means they must be on the same band (2.4 GHz, 915/868 MHz or 433 MHz). In the case of 915/868 MHz, also check that the LoRa chipsets are [compatible](SX126x_SX127x_INCOMPATIBILITY.md).  For example, the FrSky R9 series and ELRS 900 MHz receivers are not compatible with the MatekSys 900 MHz hardware.
-- Be sure you have flashed the correct [firmware file](https://github.com/olliw42/mLRS#firmware-flashing).  Files which start with tx are only for modules functioning in the Tx role and files which start with rx are only for modules functioning in the receiver role.  Also, pay attention to the variants for different use cases and hardware configurations.  For example, there are 3 separate files for different hardware and use case options for the MatekSys mR900-30 when used as a Tx module and a 4th file for when the same hardware is used as a Rx.  A module with rx firmware flashed won't connect with any other module with rx firmware flashed and tx firmware won't connect with tx firmware.
+- Be sure you have flashed the correct [firmware file](https://github.com/olliw42/mLRS#firmware-flashing).  Files which start with tx are only for modules functioning in the Tx role and files which start with rx are only for modules functioning in the receiver role.  Also, pay attention to the variants for different use cases and hardware configurations.  For example, there are 3 separate files for different hardware and use case options for the MatekSys mR900-30 when used as a Tx module and a 4-th file for when the same hardware is used as a receiver.  A module with rx firmware flashed won't connect with any other module with rx firmware flashed and tx firmware won't connect with tx firmware.
```

Comment:
'4th' looks correct to me here.  Haven't seen '4-th' before.

Reply:
okay ... I just find it very hard to read ... but I will revert :)

Reply:
grabbed the opportunity to also make all parameters being links (like most were)

Reply:
I would probably prefer 'fourth' and then anything > 10 I would switch to 11th, 12th, etc.